packages/web-components/fast-components/src/text-field/fixtures/text-field.html

```diff
@@ -97,4 +97,10 @@ <h4>Visual vs audio label</h4>
     <!-- With aria-label -->
     <h4>With aria-label</h4>
     <fast-text-field aria-label="Text field with aria-label"></fast-text-field>
+
+    <form onsubmit="console.log('form submitted!');">
```

Comment:
not sure if I should leave this in the fixture or not

Reply:
@marjonlynch - if we need a method of testing, I'd recommend using an `alert()` rather than console.log in this case so there is a visible indication.

Reply:
made it an alert instead of console.log